### PR TITLE
Multiple small changes

### LINKED ISSUE #15: Modelisation problem: scales without named anchors
For instance a junction J between A and B.

The junction starts on A at the milestone 12+100 and is 800m long.

There is no named anchor on that junction. If each lane have their own scale, the scale J won’t have a named anchor. We currently do not support it.
If there was a single scale for the 3 traversals, there is no problem.

Possible solution: add a third notion of anchor to named and unamed. Unamed with a measure to identify the start and the offset

### PR DESCRIPTION
**Pull request by Tristramg**
- The position on the curve is [0,1]. This reduces ambiguity about the unit used, and simplifies mental model
- Name of anchors are optional: allows to set the end of curve to allow computations when there is only one anchor
- Coordinates of anchors are optional: allows to handle #15 by creating virtual anchors
- Minor stuff (metadata, verbosity)

Closes #15

### CODE REVIEW

File: src/builder.rs [hunk 1]
```diff
@@ -115,29 +120,53 @@ impl<'fbb> Builder<'fbb> {
...
         self.anchors
             .push(Anchor::create(&mut self.fbb, &anchor_arg));
-        self.anchors_coordinates.push(coord);
+        self.temp_anchors.push(AnchorPosition::Geographical(coord));
 
-        self.anchors.len() - 1
+        self.temp_anchors.len() - 1
+    }
+
+    /// A a new [`Anchor`] based on its position along the curve.
```
Comment (louisgreiner) [inline src/builder.rs:148]: Nit: 1 "A" is enough :p

File: src/lrm_scale.rs [hunk 2]
```diff
@@ -127,6 +127,7 @@ pub struct LrmScale {
 impl LrmScale {
     /// Locates a point along a `Curve` given an [Anchor] and an `offset`,
     /// which might be negative.
+    /// The `[CurvePosition]` is between 0.0 and 1.0
```
Comment (louisgreiner) [inline src/lrm_scale.rs:130]: I would precise "... both included" ? or else if not

**Review by louisgreiner — APPROVED** [review]
Left a few comments on documentation, but in general we will confirm the documentation in a later work. Lgtm !

File: python/src/lib.rs [hunk 3]
```diff
@@ -252,7 +252,16 @@ impl Lrs {
             .map_err(|e| PyTypeError::new_err(e.to_string()))
     }
 
-    /// Given two [`LrmScaleMeasure`]s, return a range of [`LineString`].
+    /// bla
+    pub fn locate_point(&self, lrm_index: usize, measure: &LrmScaleMeasure) -> PyResult<f64> {
+        //self.lrs.lrs.traversals[lrm_index].curve.
```
[the remark this answers is not in the document]
Comment (Tristramg) [reply python/src/lib.rs:257]: :sweat_smile:

**Comment by Tristramg** [comment]
Thank you for the review, I've integrated all the suggestions

**Pull request merged**